Task1CSharp/Program.cs

```diff
@@ -80,5 +77,32 @@ public static void WriteFiguresToFile(string WRITE_PATH ,List<IFigure> list)
                 }
             }
         }
+
+        /// <summary>
+        /// Writes sorted by area list to file "WRITE_PATH_FILE1"
+        /// </summary>
+        /// <param name="list"></param>
+        public static void SortByArea(List<IFigure> list)
```

Comment:
It is bad manner to do many things in on method (sort and write in file) and change parameter(list). Make method which rerutn sorted list.